newtab.js

```diff
@@ -1141,7 +1143,7 @@ var themes = {
 		highlight_font_color: '#ffff80',
 		shadow_color: '#ff80c0'
 	},
-	Midnight: {
+	Middark: {

```

Comment:
```suggestion
	Midnight: {
```

Oups, that was an accident here, I called the `dark_theme` a `night_theme` at first --> replaced too much here 😅